src/config/ch.sbb.netzkarte/index.js

```diff
@@ -187,11 +187,10 @@ export const punctuality = new Layer({
 punctuality.children = [
   new TralisLayer({
     isUpdateBboxOnMoveEnd: true,
-    name: 'ch.sbb.puenktlichkeit-nv',
+    name: 'ch.sbb.puenktlichkeit-gondola',
     visible: false,
     tenant: 'sbb',
-    regexPublishedLineName: '^(S|R$|RE|PE|D|IRE|RB|TER)',
-    group: 'ch.sbb.punctuality',
+    filter: ({ properties }) => properties?.type === 'gondola',
```

Comment:
Ithink you can use the `mots`, to filter mot  instead it will avoid receiving all the vehicles events.

https://github.com/geops/mobility-toolbox-js/blob/master/src/common/mixins/RealtimeLayerMixin.ts#L177

mots: ['gondola']